src/main/resources/db/changelog/changelog.sql

```diff
@@ -267,3 +267,46 @@ CREATE INDEX `Accounts-SynapseUserId-Index` ON `Accounts`(`synapseUserId`);
 ALTER TABLE `AccountRoles`
 MODIFY COLUMN `role` enum('DEVELOPER','RESEARCHER','ADMIN','TEST_USERS','WORKER','SUPERADMIN') NOT NULL;
 
+-- changeset bridge:13
+
+CREATE TABLE `Assessments` (
+  `guid` varchar(255) NOT NULL,
+  `appId` varchar(255) NOT NULL,
+  `identifier` varchar(255) NOT NULL,
+  `revision` int(10) unsigned NOT NULL,
+  `ownerId` varchar(255),
+  `title` varchar(255) NOT NULL,
+  `summary` text,
+  `validationStatus` text,
+  `normingStatus` text,
+  `osName` varchar(255) NOT NULL,
+  `originGuid` varchar(255),
+  `customizationFields` text,
+  `links` text,
+  `createdOn` bigint(20) unsigned DEFAULT NULL,
+  `modifiedOn` bigint(20) unsigned DEFAULT NULL,
+  `deleted` tinyint(1) DEFAULT '0',
+  `version` bigint(10) unsigned NOT NULL DEFAULT 0,
+  PRIMARY KEY (`guid`),
+  UNIQUE KEY (`appId`, `identifier`, `revision`),
+  CONSTRAINT FOREIGN KEY (`originGuid`) REFERENCES `Assessments` (`guid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+CREATE INDEX `Assessments-AppId` ON `Assessments`(`appId`);
+CREATE INDEX `Assessments-AppId-Guid` ON `Assessments`(`appId`, `guid`);
```

Comment:
If guid is a primary key, what does the appId-guid index do?

Reply:
We prevent queries across app contexts by including app ID in the DAO queries, and as a result, we want to index on this as well as the key. I liked this approach because I don't have to create an object to represent a compound identifier, which is a lot of Hibernate-specific boilerplate code.